### DISCUSSION TITLE
The force information from CP2K output can not be read when using dpdata

### DISCUSSION THREAD

**Discussion by yunjian-hash** [deepmodeling/deepmd-kit#1054] [Q&A]
Dear developers,

I want to use the dpdata to obtain the inital data in 'deepmd' folders from CP2K output, but the force.raw is always empty. So, how to obtain the initial data with the output from cp2k.

![b35ee5492dde862267b0a04357bef7f](https://user-images.githubusercontent.com/75162924/131215848-52200972-21c5-4242-ba6f-cdb83b731265.png)

![0cf7d290fba6cc62781e5e1b249a036](https://user-images.githubusercontent.com/75162924/131215760-c8379bca-3309-444e-9086-e77c40854ca1.png)

![08e1bf84160fb7d2db5991e9bd5913d](https://user-images.githubusercontent.com/75162924/131215841-960bce5c-30a0-4f86-b27b-e6804c6925d6.png)

**Comment by njzjz** [comment]
@robinzyb Could you take a look?

**Comment by yunjian-hash** [comment]
Thanks I have got answers from our Wechat group

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by lizhiqiang778** [reply]
> Thanks I have got answers from our Wechat group Get Outlook for iOS<https://aka.ms/o0ukef>
> […](#)
> ________________________________ From: Jinzhe Zeng ***@***.***> Sent: Saturday, September 4, 2021 10:55 AM To: deepmodeling/deepmd-kit Cc: Li Yunjian; Author Subject: Re: [deepmodeling/deepmd-kit] The force information from CP2K output can not be read when using dpdata (#1054) @robinzyb<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_robinzyb&d=DwMCaQ&c=KXXihdR8fRNGFkKiMQzstu-8MbOxd1NuZkcSBymGmgo&r=HogfIrfCNEAkSESYG4Drrw&m=iSREmdXTW3DsFGIxWzY_GrFycj1Pv6B3DkNO1XvS0dc&s=yM74rWIxbpmW2WdCFgNCsGzuU21F-V5HnJDWtgBytXw&e=> Could you take a look? — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_deepmodeling_deepmd-2Dkit_discussions_1054-23discussioncomment-2D1279494&d=DwMCaQ&c=KXXihdR8fRNGFkKiMQzstu-8MbOxd1NuZkcSBymGmgo&r=HogfIrfCNEAkSESYG4Drrw&m=iSREmdXTW3DsFGIxWzY_GrFycj1Pv6B3DkNO1XvS0dc&s=lm3GZ-ZE9cUvOlteW6p__BorX2xnLh_r3WrTLwPMfLw&e=>, or unsubscribe<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AR5OKLH7Q36DC4CPJTKEXN3UAGDCHANCNFSM5C7ADUYA&d=DwMCaQ&c=KXXihdR8fRNGFkKiMQzstu-8MbOxd1NuZkcSBymGmgo&r=HogfIrfCNEAkSESYG4Drrw&m=iSREmdXTW3DsFGIxWzY_GrFycj1Pv6B3DkNO1XvS0dc&s=RrnKk_SI2EyEFg2KCIyLEP322kvVrgFlfBHX-a6gD98&e=>. Triage notifications on the go with GitHub Mobile for iOS<https://urldefense.proofpoint.com/v2/url?u=https-3A__apps.apple.com_app_apple-2Dstore_id1477376905-3Fct-3Dnotification-2Demail-26mt-3D8-26pt-3D524675&d=DwMCaQ&c=KXXihdR8fRNGFkKiMQzstu-8MbOxd1NuZkcSBymGmgo&r=HogfIrfCNEAkSESYG4Drrw&m=iSREmdXTW3DsFGIxWzY_GrFycj1Pv6B3DkNO1XvS0dc&s=ABffrMkRpsYEs4T00ybA6pqXHGmT7Szme8lGBEPwamQ&e=> or Android<https://urldefense.proofpoint.com/v2/url?u=https-3A__play.google.com_store_apps_details-3Fid-3Dcom.github.android-26referrer-3Dutm-5Fcampaign-253Dnotification-2Demail-2526utm-5Fmedium-253Demail-2526utm-5Fsource-253Dgithub&d=DwMCaQ&c=KXXihdR8fRNGFkKiMQzstu-8MbOxd1NuZkcSBymGmgo&r=HogfIrfCNEAkSESYG4Drrw&m=iSREmdXTW3DsFGIxWzY_GrFycj1Pv6B3DkNO1XvS0dc&s=wsQCl5ZmI08Xq5d94T4Kwd30bPOkrtgYIK4bQDq3d7s&e=>.

hi , i met the same question with you . could you please tell me how you solve this ?
